src/muya/lib/parser/rules.js

```diff
@@ -10,9 +10,10 @@ export const beginRules = {
 }
 
 export const inlineRules = {
-  'backlash': /^(\\)([\\`*{}\[\]()#+\-.!_>~:\|\<\>]{1})/,
+  'backlash': /^(\\)([\\`*{}\[\]()#+\-.!_>~:\|\<\>$]{1})/,
```

Comment:
Can you please separate original, patched and new rules like [here](https://github.com/marktext/marktext/blob/0a3e05c0c12a32a198a936d5e1c0901d160f01ae/src/muya/lib/parser/marked/inlineRules.js#L32)?